gnovm/stdlibs/std/banker.go

```diff
@@ -88,10 +89,30 @@ func X_bankerTotalCoin(m *gno.Machine, bt uint8, denom string) int64 {
 
 func X_bankerIssueCoin(m *gno.Machine, bt uint8, addr string, denom string, amount int64) {
 	// gno checks for bt == RealmIssue
-	m.Context.(ExecContext).Banker.IssueCoin(crypto.Bech32Address(addr), denom, amount)
+
+	// check origin denom format
+	matched, err := regexp.MatchString("[a-z][a-z0-9]{2,15}", denom)
+	if err != nil || !matched {
+		m.Panic(typedString("invalid denom format, must be [a-z][a-z0-9]{2,15}"))
+		return
+	}
```

Comment:
Can we keep the regexes separated as constants, especially if they're repeated in the error message like this?

Reply:
3838b51d0e8930a0a45fe663472866bd4a1446c5